amplify/backend/api/colonycdapp/schema.graphql

```diff
@@ -8,492 +8,967 @@ input AMPLIFY {
+"""
 input UpdateExtensionByColonyAndHashInput {
+  """The unique identifier for the Colony."""
   colonyId: ID!
+  """The hash of the extension to be updated."""
   hash: String!
+  """A flag to indicate whether the extension is deprecated."""
   isDeprecated: Boolean
+  """A flag to indicate whether the extension is deleted."""
   isDeleted: Boolean
+  """A flag to indicate whether the extension is initialized."""
   isInitialized: Boolean
+  """The version of the extension."""
   version: Int
+  """The Ethereum address of the user who installed the extension."""
   installedBy: String
+  """The timestamp when the extension was installed."""
   installedAt: AWSTimestamp
 }
 
+"""
+Input data to store the latest available version of the core Colony contract and available extensions
+
+The extension hash is generated like so: `keccak256(toUtf8Bytes(extensionName))`, where `extensionName` is the name of the extension contract file in the Colony Network (e.g. `VotingReputation`)
+"""
 input SetCurrentVersionInput {
+  """COLONY for the Colony contract, extension hash for extensions"""
   key: String!
+  """Latest available version"""
   version: Int!
 }
 
+"""
+Return type for tokens gotten from DB or from chain.
+"""
 type TokenFromEverywhereReturn {
+  """List of tokens found"""
   items: [Token]
 }
 
+"""
+Input data for retrieving a user's reputation within the top domains of a Colony.
+"""
 input GetReputationForTopDomainsInput {
+  """The Ethereum wallet address of the user."""
   walletAddress: String!
+  """The Ethereum address of the Colony."""
   colonyAddress: String!
+  """The root hash of the reputation tree at a specific point in time."""
   rootHash: String
 }
 
+"""
+Input data for retrieving a user's token balance for a specific token.
+"""
 input GetUserTokenBalanceInput {
+  """The Ethereum wallet address of the user."""
   walletAddress: String!
+  """The Ethereum address of the token."""
   tokenAddress: String!
 }
 
+"""
+A type representing a user's reputation within a domain.
+"""
 type UserDomainReputation {
+  """The integer ID of the Domain within the Colony."""
   domainId: Int!
+  """The user's reputation within the domain, represented as a percentage."""
   reputationPercentage: String!
 }
 
+"""
+A return type that contains an array of UserDomainReputation items.
+"""
 type GetReputationForTopDomainsReturn {
+  """An array of UserDomainReputation items."""
   items: [UserDomainReputation!]
 }
 
+"""
+A return type representing the breakdown of a user's token balance.
+
+Definitions:
+* Inactive: balance of a token that is in a users wallet but can't be used by the Colony Network (e.g. for governance)
+* Locked: balance of a token that is staked (e.g. in motions)
+* Active: balance that is approved for the Colony Network to use (e.g. for governance)
+* Pending: [needs input]
```

Comment:
This needs some input, I can't remember what this is used for.

Reply:
@jakubcolony worked on this recently so may be able to help?

Reply:
It's `pendingBalance` property from `userLock`. Can't say anything more. Some pieces from the docs I found in the GitHub org:

> `pendingBalance` Tokens that have been sent to them, but are inaccessible until all locks are cleared and then these tokens are claimed"

> // Pending balance (from failed transfers), can claim with a forced deposit